components/analytics-tools/org.wso2.carbon.analytics.tools.backup/src/main/java/org/wso2/carbon/analytics/tools/backup/AnalyticsDataBackupTool.java

```diff
@@ -320,5 +336,170 @@ private static AnalyticsSchema readTableSchema(String basePath) throws IOExcepti
             }
         }
     }
-    
+
+    /**
+     * Backs up the file system to Local
```

Comment:
End the main comment with a full stop. Fix in all the places, including class level comments etc..